src/hooks/useFullScreenAd.ts

```diff
@@ -73,7 +73,7 @@ export function useFullScreenAd<
           setState({ isLoaded: true });
           break;
         case AdEventType.OPENED:
-          setState({ isOpened: true });
+          setState({ isOpened: true, isLoaded: false });
```

Comment:
Please align with: https://github.com/invertase/react-native-google-mobile-ads/blob/main/src/ads/MobileAd.ts#L98
Where we reset loaded after closing.